pddl-examples/blocksdomain.pddl

```diff
@@ -0,0 +1,17 @@
+(define (domain adl-blocksworld)
```

Comment:
No, the pddl examples folder is still there. I removed the planner folder. The pddl examples were needed for testing dockerization of the planner. Since they are already in ff-metric-docker/tests, we can as well delete them